modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/restapi/ContentSearchTestCase.java

```diff
@@ -123,15 +125,16 @@
             //search term : UnifiedSearchFeature, created api has this in description filed
             getStoreAPIs.setHeader("Authorization", "Bearer " + accessToken);
             HttpResponse storeResponse = client.execute(getStoreAPIs);
-            if (getResultCount(storeResponse) == 1) {
+            responseString = getResponseBody(storeResponse);
+            if (getResultCount(responseString) == 1) {
                 Assert.assertTrue(true);
                 break;
             } else {
                 if (i == retries) {
-                    Assert.fail("Basic content search in store failed. Received response : " + storeResponse);
+                    Assert.fail("Basic content search in store failed. Received response : " + responseString);
                 } else {
-                    log.warn("Basic content search in store failed. Received response : " + storeResponse + " Retrying...");
-                    Thread.sleep(3000);
+                    log.warn("Basic content search in store failed. Received response : " + responseString + " Retrying...");
```

Comment:
Length of the line

Reply:
fixing in https://github.com/wso2/product-apim/pull/5009